### PR TITLE
fixing CLA issie

### PR DESCRIPTION
**Pull request by Kjaroszew**
I want to merge this change because... i want to fix issue with CLA for translations tests

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://stable.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5500 Update tests for translations](https://app.clickup.com/t/2549495/SALEOR-5500)

**Comment by karola312** [comment]
Close previous PR -> SALEOR-5500-Test-Update-Translation, SALEOR-5500-Test-Update-Translation-for-3.1. Resolve conflicts.

File: cypress/integration/translations.js [hunk 1]
```diff
@@ -7,10 +7,10 @@ import {
...
-import { deleteCategoriesStartsWith } from "../../support/api/utils/catalog/categoryUtils";
-import filterTests from "../../support/filterTests";
-import { updateTranslationToCategory } from "../../support/pages/translationsPage";
+} from "../support/api/requests/Category";
+import { deleteCategoriesStartsWith } from "../support/api/utils/catalog/categoryUtils";
+import filterTests from "../support/filterTests";
+import { updateTranslationToCategory } from "../support/pages/translationsPage";
 
 filterTests({ definedTags: ["all"], version: "3.0.0" }, () => {
   describe("As an admin I want to manage translations", () => {
```
Comment (eCzerniak) [inline cypress/integration/translations.js:16]: In line 17 `const startsWith = "Translations";` cause that tests finds this word in first row and they fail.
image

**Review by eCzerniak — CHANGES_REQUESTED** [review]

File: cypress/integration/translations.js (hunk 1 above)
Comment (Kjaroszew) [reply cypress/integration/translations.js:16]: changed form Translations to TestTranslations

**Review by eCzerniak — APPROVED** [review]

**Review by michalina-graczyk — APPROVED** [review]

**Pull request merged**